rcl_logging_spdlog/README.md

```diff
@@ -2,6 +2,16 @@
 
 Package supporting an implementation of logging functionality using `spdlog`.
 
+The `rcl_logging` interface allows [rcl_logging_spdlog/logging_interface.h](include/rcl_logging_spdlog/logging_interface.h):
+ - initialize
```

Comment:
@ahcorde should we speak about this specific `spdlog` implementation instead? I'd think `rcl_logging` documentation already describes the interface in full.